extensions/amp-analytics/0.1/test/test-linker-manager.js

```diff
@@ -628,6 +628,30 @@ describes.realWin('Linker Manager', {amp: true}, env => {
       });
     });
 
+    it('should not rewrite url if href is fragment', () => {
+      const lm = new LinkerManager(ampdoc, config, /* type */ null, element);
+      return lm.init().then(() => {
```

Comment:
drive-by indentation nit: `async/await` 🚗

```js
it('should not rewrite url if href is fragment', async () => {
  const lm = new LinkerManager(ampdoc, config, /* type */ null, element);
  await lm.init();
	// Using a real anchor el here because of a.href getter will actually
  // return the full url, not just the fragment.
  const a = doc.createElement('a');
  // ...
});
```